docs/1 - authentication.md

```diff
@@ -206,3 +236,18 @@ if ($user->tokenCant('forums:manage'))
 }
 ```
 
+## Pipeline Handler
+
+The Pipeline handler is a unique case. It does not have any of its own logic. Instead, it runs the 
+desired method called through each of the other specified handlers, until it either gets a successful 
+result or they all fail. When it fails it only returns the result from the last handler, so you may want
```

Comment:
Is there a way this could be abstracted even more such that pipeline could eventually handle dual-authentication? i.e. right now the handlers are `OR` but with 2FA they would be `AND`.

Reply:
That's a good question. That is coming up on my list very shortly to figure out how to handle 2FA, activators, etc. I'll think on that.